src/style.css

```diff
@@ -2,2 +1,83 @@
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
```

Comment:
It's good that you're using a specific font for your body tag, but remember to use fallback fonts. This is an alternative font-family in case the main one doesn't work. For example: `font-family: Roboto, Arial, sans-serif;`